zipkin-storage/mysql-v1/src/test/java/zipkin2/storage/mysql/v1/MySQLExtension.java

```diff
@@ -113,7 +113,7 @@ int port() {
   // mostly waiting for https://github.com/testcontainers/testcontainers-java/issues/3537
   static final class MySQLContainer extends GenericContainer<MySQLContainer> {
     MySQLContainer() {
-      super(parse("ghcr.io/openzipkin/zipkin-mysql:2.23.2"));
+      super(parse("ghcr.io/openzipkin/zipkin-mysql:2.24.3"));
```

Comment:
this moves to the last published docker tag (well ignoring the accidental v3 one..). Doing this gets the recent cut of test images, even though they are a bit behind still. I'll update those later.

Reply:
I refreshed these